jans-fido2/model/src/main/java/io/jans/fido2/model/common/PublicKeyCredentialParameters.java

```diff
@@ -0,0 +1,42 @@
+package io.jans.fido2.model.common;
+
+public class PublicKeyCredentialParameters {
+    private int alg;
+    private String type = PublicKeyCredentialType.PUBLIC_KEY.getKeyName();
+
+    public PublicKeyCredentialParameters() {
+    }
+
+    public PublicKeyCredentialParameters(int alg, String type) {
```

Comment:
type can be set by default to "public-key"

Reply:
done